exec/memory/base/src/main/java/org/apache/drill/exec/memory/Accountant.java

```diff
@@ -80,6 +119,23 @@ public Accountant(Accountant parent, long reservation, long maxAllocation) {
   }
 
   /**
+   * Request lenient allocations: allows exceeding the allocation limit
+   * by the configured grace amount. The request is granted only if strict
+   * limits are not required.
+   *
+   * @param enable
+   */
+  public boolean setLenient() {
```

Comment:
Why "set" returns a boolean ? Should better have a separate "get".

Reply:
Returned for testing, which is the only place that really cares about whether the leniency request was granted. I should, however, add a Javadoc comment to that effect.